addons/mass_mailing/static/src/scss/mass_mailing.ui.scss

```diff
@@ -344,19 +344,26 @@ body.editor_enable.o_basic_theme.o_in_iframe {
             display: block;
         }
     }
-}
-
-.o_mass_mailing_iframe #oe_snippets.o_loaded {
-    transition: none;
-    transform: translateX(0);
+    #oe_snippets.o_loaded {
+        transform: translateX(0);
+
+        > .o_we_customize_panel .oe-toolbar {
+            #style.dropup, #font-size.dropup {
+                > .dropdown-menu {
+                    overflow-y: scroll;
+                    max-height: 200px;
+                }
+            }
 
-    > .o_we_customize_panel .oe-toolbar {
-        #style.dropup, #font-size.dropup {
-            > .dropdown-menu {
-                overflow-y: scroll;
-                max-height: 200px;
+            // Popper.js manipulates transform and position of the dropdown, and in mass_mailing, its calculations are wrong.
+            // These !important rules stops it from doing so, by unsetting the automatic rules generated by Popper.
```

Comment:
Thanks, this is a lot better :+1: Only one thing: it should be "stop" instead of "stops" (here and in the commit message) but I'm not going to block the PR for this.